poetry.lock

```diff
@@ -386,7 +386,7 @@ python-versions = "*"
 name = "six"
 version = "1.16.0"
 description = "Python 2 and 3 compatibility utilities"
-category = "main"
+category = "dev"
 optional = false
 python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*"
```

Comment:
I'd expect all of this — and actually all occurrences of `six` spread around this lock file — to be removed once I told poetry to do so — a.k.a. `poetry remove six`. 🤷🏻‍♂️